packages/core-mobile/app/services/earn/utils.ts

```diff
@@ -123,7 +123,9 @@ export const getAvailableDelegationWeight = (
   const nAvax = getStakingConfig(isDeveloperMode).MaxValidatorStake
   const maxValidatorStake = new TokenUnit(nAvax, 9, 'AVAX')
   const maxWeight = calculateMaxWeight(maxValidatorStake, validatorWeight)
-
+  if (maxWeight.lt(validatorWeight)) {
+    return maxWeight.sub(delegatorWeight)
+  }
```

Comment:
if validatorWeight is more than then maxWeight, don't subtract maxWeight from validatorWeight.

Reply:
https://ava-labs.atlassian.net/browse/CP-9539